### DISCUSSION TITLE
Keyboard or remote home button

### DISCUSSION THREAD

**Discussion by matthewpritchett** [jellyfin/jellyfin-desktop#555] [Q&A]
Just curious if there’s a shortcut to go to the home screen from anywhere inside the app via the keyboard or a remote? I’ve tried everything I can think of and only can ever go back one page with ESC.

thanks!

**Comment by MaxBlack84840** [comment]
Hello, I discovered Jeffyfin and I love it. But I use it on QNAP in HDMI. A solution for using the remote control? After that I would be happy to use Jellyfin as a replacement for KODI. Many thanks to the developer.
